source/tutorials/moving-pictures.rst

```diff
@@ -467,19 +469,24 @@ We're also often interested in performing a differential abundance test at a spe
      --p-level 6 \
      --o-collapsed-table gut-table-l6.qza
 
-   qiime composition add-pseudocount \
+   qiime composition ancombc \
      --i-table gut-table-l6.qza \
-     --o-composition-table comp-gut-table-l6.qza
-
-   qiime composition ancom \
-     --i-table comp-gut-table-l6.qza \
      --m-metadata-file sample-metadata.tsv \
-     --m-metadata-column subject \
-     --o-visualization l6-ancom-subject.qzv
+     --p-formula 'subject' \
+     --o-differentials l6-ancombc-subject.qza
+
+   qiime composition da-barplot \
+     --i-data l6-ancombc-subject.qza \
+     --p-significance-threshold 0.001 \
+     --p-level-delimiter ';' \
+     --o-visualization l6-da-barplot-subject.qzv
 
 .. question::
-   Which genera differ in abundance across subject? In which subject is each genus more abundant?
+   1. Which genera is most enriched? Which is most depleted?
+   2. Do we see more differentially abundant features in the ``da-barplot-subject.qzv`` visualization, or in the ``l6-da-barplot-subject.qzv`` visualization? Why?
 
+.. g__Parabacteroides (enriched), g__Paraprevotella (depleted)
+.. We see more in the original vs. the collapsed table, which is reasonable since we are collapsing at L6 and thus losing some resolution.
```

Comment:
```suggestion
.. We see more differential abundant features in the original compared the collapsed table, which is reasonable since we are collapsing at the genus level and thus losing some resolution. However, collapsing at level 6 may allow us to investigate patterns that aren't present when looking at ASVs
```